ch03.asciidoc

```diff
@@ -289,17 +289,21 @@ While this method could be fully executed faster than the good and the bad way t
 === Invoices
 
 Every payment on the Lightning Network starts with a person who wants to receive bitcoins issuing an invoice.
+An invoice is nothing more than a very simple payment instruction containing information such as a recipient and an amount.
 The main reason for this process is that it helps to make the payment process over a path of payement channels atomic.
-Atomic means that no node on the path can decide to take the money that is being routed and stop the routing process.
-The payment will either be transferred successfully through the path of nodes or will not be delivered.
+Atomic means that no node on the path can decide to take the money that is being routed or stop the routing process.
+The payment will either be transferred successfully through the complete path of nodes or will not be delivered at all.
 There are no such things as a partial payment or a half successful payment.
-While Lightning Nodes usually use the encrypted communication channels over the peer to peer network to exchange information, invoices are being transferred via a second communication channel.
-This could be via a Webservice or Email.
-Invoices are usually encoded either as long bech32 strings or as QR codes which enables them to be easily scanned by smartphones.
-Obviously the invoices contains the amount of bitcoin that is requested and a signature of the payee.
+While Lightning Nodes usually use the encrypted communication channels over the Lightning peer-to-peer network to exchange information, invoices are being transferred via a second communication channel.
+The payee has to pass the invoice to the payer via any arbitrary means.
+This could be done via a Webservice, an instant message on a chat app, an SMS or an Email.
+Invoices are usually encoded either as long bech32-encoded strings or as QR codes which enables them to be easily scanned by smartphones.
+The invoices contains the amount of bitcoin that is requested and a signature of the payee.
 The later is used to extract the address of the payee so that the payer knows whom to send the money.
 Besides some other meta data the most important but not quite obvious data in the invoice is a Payment Hash.
 
+Did you notice how this contrasts with Bitcoin and how different language terms are used? In Bitcoin, the recipient passes an address to the sender. In Lightning, the payee creates an invoice and sends an invoice to the payer. In Bitcoin, the payer sends funds to an address. In Lightning, the payer pays an invoice and it gets routed to the payee. Bitcoin is based on the concept of an "address", and Lightning is a payment network and based on the concept of an "invoice".
```

Comment:
while we are at it we could also stress that on bitcoin we have transactions while on lightning we have payments

Reply:
I thought about it, but put it into the "Differences to Bitcoin" section. 
I was afraid it would get repetitive.
Still, it is just 1 sentence more. 
I added it. 
See commit # 3 in 9a737b111b96cd202f645b8e1a190d75e40ffd82